packages/widget-message-meet/src/index.html

```diff
@@ -5,6 +5,10 @@
     <meta charset="utf8"/>
   </head>
   <body>
-    <main id="main" data-toggle="spark-message-meet"></main>
+    <main id="main"
+      data-toggle="spark-message-meet"
+      data-access-token="<%= process.env.CISCOSPARK_ACCESS_TOKEN %>"
```

Comment:
Yea, in webpack I clear those env vars before it hits the build. That way if there are any places in the JS that are using those env vars they won't get distributed either.